src/shared/varlink-io.systemd.Network.c

```diff
@@ -40,11 +40,16 @@ static VARLINK_DEFINE_METHOD(
                 VARLINK_DEFINE_INPUT(InterfaceName, VARLINK_STRING, VARLINK_NULLABLE),
                 VARLINK_DEFINE_OUTPUT_BY_TYPE(Neighbors, LLDPNeighborsByInterface, VARLINK_ARRAY));
 
+static VARLINK_DEFINE_METHOD(
+                SetPersistentStorage,
+                VARLINK_DEFINE_INPUT(Ready, VARLINK_BOOL, 0));
+
```

Comment:
might be nice to do:

```c
static VARLINK_DEFINE_ERROR(StorageIsReadOnly);
```

here, and then list in in the interface, and then fire that in the EROFS case above